buildSrc/src/main/kotlin/Pom.kt

```diff
@@ -0,0 +1,28 @@
+import org.gradle.api.publish.maven.MavenPublication
+
+fun MavenPublication.defaultPom() = pom {
```

Comment:
Couldn't this also be inlined? Since it's only used once and the content is quite specific, it's probably not necessary to pull this into a separate method.

Reply:
It's for readability, the pom is quite large so I'd rather have it in a separate file so the build.gradle.kts files are easier to read.